SwiftUIDZ/SwiftUIDZ/ContentView.swift

```diff
@@ -8,9 +8,168 @@
 import SwiftUI
 
 struct ContentView: View {
+    
+    private enum Constants {
+        static let wifiState = ["Подключено", "Не подключено"]
+        static let onOfState = ["Вкл.", "Выкл."]
+        static let navigationTitle = "Настройки"
+        static let vpnName = "VPN"
+        static let swiftImageSystemName = "swift"
+        static let modemName = "Режим модема"
+        static let modemImageSystemName = "personalhotspot"
+        static let mobileName = "Сотовая связь"
+        static let antennaImageSystemName = "antenna.radiowaves.left.and.right"
+        static let bluetoothName = "Bluetooth"
+        static let wifiName = "Wi-Fi"
+        static let wifiImageSystemName = "wifi"
+        static let airplaneName = "Авиарежим"
+        static let airplaneImageSystemName = "airplane"
+        static let iosSettings = "iOS 16.2: уже доступно"
+        static let offerSettings = "Предложения Apple ID"
+        static let badgeOne =  "1"
+        static let badgeThree = "3"
+        static let nameSubTitle = "Apple ID, iCloud, контент и покупки"
+        static let nameTitle = "MyName"
+        static let personImageSystemName = "person.fill"
+    }
+    
+    @State private var isOnAirplaneMode = false
+    @State private var wifiSection = 0
+    @State private var bluetoothSection = 0
+    @State private var modemSection = 0
+    @State private var isOnVPN = false
+    
     var body: some View {
-        Text("Hello, world!")
-            .padding()
+        NavigationView {
+            Form {
+                Section {
+                    NavigationLink(destination: Text("")) {
```

Comment:
```suggestion
                    NavigationLink(destination: Text("")) {
```
можно "" в константы